### PR TITLE
Adds the quarterly newsletter

### PR DESCRIPTION
**Pull request by gboone**
Updates from PI10!

### CODE REVIEW

File: content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+### What’s new
+
+#### Use the dashboard to invite new users to your team
+
+Managing users in your organization and granting them the permissions they need should be a straightforward process so your team can get to work building your app on cloud.gov. To make it easier for you, we built new features for inviting new users and giving them roles within your organization using the dashboard. With these changes, even the least technical people on your team can invite new folks and manage their role without involving cloud.gov support.
+
+<img src="/img/invite-users.png" alt="The user invitation screen with notification" width="624" height="272" />
+
+Being able to manage your users from the dashboard will also allow you to have finer control over who has access to different parts of your system without having to involve your technical staff. Our users requested this functionality, so if you get a chance to use it please let us know how it’s working for you. Watch for more developments like this in the future.
```
Comment (brittag) [inline content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md]: We removed this second paragraph (and adjusted the earlier one) a bit [in the update sent out](http://us9.campaign-archive2.com/?u=6f1977de9eff4c384dc8d6527&id=fbc1e98d98) - fine with me to bring it back if you'd like, just not sure if intentional. (Also _just_ noticed a typo in the update we sent - "With these changes, it's easy for even the least technical people on your team **can** invite new folks and manage their roles.")

File: content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+#### Continuous improvement
+
+Staying compliant with federal laws is crucial to us and to you, but compliant simply isn’t good enough, or fast enough, for us. That’s why [cloud.gov is designed and operated to routinely provide apps the latest security updates](https://www.google.com/url?q=https://cloud.gov/overview/overview/cloudgov-benefits/&sa=D&ust=1501263385089000&usg=AFQjCNGRQGmoDF2V5KRSRg1JaAFp_yMYYg) within a day of their release. When there’s a major vulnerability in a component managed by cloud.gov, our team makes these updates across the entire platform, so you don’t have to spend time doing it. For updates to the buildpack, operating system, or [any other component we manage](https://cloud.gov/overview/technology/responsibilities/), all that’s typically required is a [restage of your apps](https://cloud.gov/docs/getting-started/app-maintenance/#restaging-your-app), even as they scale and grow, making security straightforward and easy for you.
```
Comment (brittag) [inline content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md]: "cloud.gov is designed and operated to routinely provide apps the latest security updates" ended up with an odd Google URL link, and "making security straightforward and easy for you" is another variation from the email update.

File: content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The Federal Bureau of Investigation (FBI) recently launched their new Crime Data Explorer (CDE) on cloud.gov. 18F and the FBI partnered to make crime data more accessible to the American public. The CDE enables users to visualize national, state, and local crime trends and offers bulk datasets and an open API for more detailed views of the data. Check it out at: [https://crime-data-explorer.fr.cloud.gov/](https://crime-data-explorer.fr.cloud.gov/)
+
+<img src="/img/fbi-crime-data.png" alt="the fbi crime data explorer homepage" width="624" height="320" />
+
+#### Federal Election Commission
+
+The Federal Election Commission (FEC) recently relaunched their flagship website, fec.gov, with cloud.gov. The FEC collects financial reports for all federal elections and discloses fundraising figures to the public. By hosting FEC.gov on cloud.gov and moving its data to the cloud, the FEC anticipates saving 85% in hosting costs and is better prepared for peak traffic events. Read a case study about the FEC here: [https://cloud.gov/case-study/fec/](https://cloud.gov/overview/customer-stories/fec/) and check out their new, cloud.gov-hosted website at [https://fec.gov](https://fec.gov)
```
Comment (brittag) [inline content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md]: This has the old case study link

File: content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md (hunk 1 above)
Comment (mogul) [reply content/updates/2017-08-11-continuous-improvement-more-tools-help-use-cloud-gov.md]: I noticed that after we sent it too. :(

**Comment by gboone** [comment]
OK, patched up those inline comments. I think we're set now. I'm pretty confused how those got in there, I must have converted the markdown based on a slightly out of date version… or something. Dunno.

**Comment by brittag** [comment]
This looks fine to me now - do you want it to be dated 2017-08-11 to match the email, or should we bump the date?

**Review by brittag — APPROVED** [review]
Looks good; question about date.

**Comment by gboone** [comment]
I'm good letting the date match the email even though it'll be out of order on the /updates/ page.

**Pull request merged**